spec/lib/zipkin_b3_single_header_format_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'spec_helper'
+
+describe ZipkinTracer::B3SingleHeaderFormat do
+  let(:b3_single_header_format) { described_class.parse_from_header(b3_single_header) }
+
+  context 'child span' do
```

Comment:
add some malformed tests? Ex in brave we use

"not-a-tumor"
"b970dafd-0d95-40aa-95d8-1d8725aebe40"

Reply:
@adriancole In ruby client, we are doing malformed check in another class: 

https://github.com/openzipkin/zipkin-ruby/blob/a1b1523a5bfcde3b6d2a8213a76e76b7a0e8a088/lib/zipkin-tracer/trace.rb#L134-L150

I think we don't need to have malformed tests here but what do you think?